src/components/TheTimelapseRenderingSnackbar.vue

```diff
@@ -2,12 +2,12 @@
 
 <template>
     <div>
-        <v-snackbar v-model="boolShowDialogRunning" :timeout="-1" :value="true" fixed right bottom dark>
+        <v-snackbar v-model="boolShowDialogRunning" :timeout="-1" :value="true" fixed right bottom>
```

Comment:
I think this will be wrong, if you just delete the dark option with no condition.